app/src/main/java/com/github/se/signify/MainActivity.kt

```diff
@@ -78,7 +78,7 @@ class MainActivity : ComponentActivity() {
     val savedLanguage = sharedPreferencesLanguage.getBoolean(prefKeyLanguage, false)
 
     // Tutorial versioning logic
-    val currentTutorialVersion = 2 // Update this when the tutorial changes
+    val currentTutorialVersion = 3 // Update this when the tutorial changes
```

Comment:
I'm sure there's a more formal way to do this, but we probably won't be updating the tutorial anymore anyway sadly.